### PR TITLE
feat: Add select catalog page

### PR DESCRIPTION
**Pull request by lynnjepsen**

### CODE REVIEW

File: src/SelectPage.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        </div>
+      </div>
+    );
+  }
+}
+
+class SelectDemos extends Component {
+  constructor(props) {
+    super(props);
+    this.selects = [];
```
Comment (patrickrodee) [inline src/SelectPage.js]: Could move this out as an instance variable.

`selects = [];`

then there's no need for a constructor

File: src/SelectPage.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+          <div className='mdc-line-ripple'></div>
+        </div>
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <div>
+        {this.renderSelectVariant('Box Select', 'mdc-select--box')}
```
Comment (rlfriedman) [inline src/SelectPage.js]: Only one variant right now, so renderSelectVariant might not be needed. Is this just here to easily add more variants in the future?

File: src/App.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ import FabPage from './FabPage';
 import IconTogglePage from './IconTogglePage';
 import ImageListPage from './ImageListPage';
 import LinearProgressPage from './LinearProgressIndicatorPage';
+import SelectPage from './SelectPage';
```
Comment (rlfriedman) [inline src/App.js]: nit: Move below ListPage

File: src/SelectPage.js (hunk 2 above)
Comment (lynnjepsen) [reply src/SelectPage.js]: ya, just easier for when we add the outlined version

File: src/styles/SelectPage.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+@import "@material/select/mdc-select";
+
+.demo-select {
+  min-width: 300px;
+}
```
Comment (patrickrodee) [inline src/styles/SelectPage.scss]: Nit: add a trailing new line

**Pull request merged**